src/main/webapp/pages/projects/settings/pages/delete.html

```diff
@@ -1,44 +1,95 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      data-layout-decorate="~{projects/settings/_base}">
-<head>
+<html
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+  data-layout-decorate="~{projects/settings/_base}"
+>
+  <head>
     <title th:text="#{project.settings.page.delete}">Basic</title>
-</head>
-<body>
-<th:block layout:fragment="settings-content">
-    <div class="row">
+  </head>
+  <body>
+    <th:block layout:fragment="settings-content">
+      <div class="row">
         <div class="col-md-12">
-            <h2 class="h2-muted" th:text="#{project.settings.page.delete}">_Delete Project_</h2>
-            <div class="alert alert-warning" role="alert">
-                <p><strong th:text="#{project.settings.delete.warning}">_Warning! Deletion of a project is permanent action!_</strong></p>
-                <p><span th:text="#{project.settings.delete.consequences}">Deleting a project will have the following consequences:</span>
-                <ul>
-                    <li th:text="#{project.settings.delete.consequences.samples}">Access to this project's samples will be removed for all users of this project.</li>
-                    <li th:text="#{project.settings.delete.consequences.copied}">Samples which have been copied to other projects will not be affected.  They will still be available in their other projects.</li>
-                    <li th:text="#{project.settings.delete.consequences.reference}">Reference files associated with this project will no longer be accessable.</li>
-                    <li th:text="#{project.settings.delete.consequences.associated}">Associated projects will no longer have access to this project's samples.</li>
-                    <li th:text="#{project.settings.delete.consequences.analyses}">Analyses shared with this project will no longer be shared with this project's users.  Submitted analyses will still be avilable to the submitting users in their analysis list</li>
-                    <li th:text="#{project.settings.delete.consequences.ncbi}">Information about NCBI submissions from this project will no longer be accessable.  Data which has been uploaded to NCBI will still be available on NCBI's sites.</li>
-                </ul>
-                </p>
+          <h2 class="h2-muted" th:text="#{project.settings.page.delete}">
```

Comment:
This html below should no longer be required correct?

Reply:
Removed in b0bb1e4.  This is actually done in a later PR but can be done here.